service-spec/service-spec-commons/src/main/resources/properties/field-data.properties

```diff
@@ -1,2 +1,4 @@
 volumes=name
-agents=name
\ No newline at end of file
+agents=name
+containers=name
+ports=name
```

Comment:
Better not combine these with existing. strategic merge patch of hspec & hprof is different from strategic merge patch of generated k8s yamls & patch snippets. The content in this file helps to resolve the ambiguity on arrays while performing respective merge patches.